docs/tutorial/t-spark-monitoring.md

```diff
@@ -0,0 +1,350 @@
+## Monitoring the Spark Cluster
+
+By default, Spark stores the logs of drivers and executors as pod logs in the local file system, which gets lost if the pods are deleted. Spark provides us with the feature to store these logs in S3 so that they can later be retrieved and visualized.
+
+Monitoring of Spark cluster can be done in two ways,
+1. Using Spark History Server 
+2. Using Canonical Observability Stack (COS)
+
+Let's explore each one of these.
+
+### Monitoring with Spark History Server
+
+Spark History Server is a user interface to monitor the metrics and performance of completed and running Spark applications. The Spark History Server is offered as a charm in the Charmed Spark solution, which can be deployed via Juju.
+
```

Comment:
I think it would be nice to have a small breakdown of the steps to expect, similar to what you did in the next section about COS. 
E.g.
1. Set up the Spark History Server with Juju
2. Connect SHS to the S3 bucket
3. Access the web UI to view and monitor jobs

Reply:
This breakdown is provided in the paragraph that follows just after the `juju add-model` command.